SingularityUI/app/components/apidocs/ApiDocs.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import rootComponent from '../../rootComponent';
+import { refresh } from '../../actions/ui/apidocs';
+
+import SwaggerUi, {presets} from 'swagger-ui';
+import 'swagger-ui/dist/swagger-ui.css';
+
+class ApiDocs extends Component {
+
+  componentDidMount() {
+    SwaggerUi({
```

Comment:
@kwm4385 no rush, but if you get a chance I'm a bit stumped. This works locally when running with gulp serve, pointed against a local singularity or against our staging environment. But, the deployed version doesn't work. Comes back with errors of `Expected the reducer to be a function`.